src/andamio/course/100/assignment100.md

```diff
@@ -17,18 +17,22 @@ The purpose of this Assignment is to show you how it works.
 ## Step By Step:
 
 ### Step 1: Mint a Learner Token
+
 [Click here](/course-management/roles/learner/dashboard) to see your Learner Dashboard. After you connect a wallet, you can click the button that says "Mint Your Learner Token". Give it a try!
 
 ### Step 2: Commit to an Assignment
+
 1. Make sure that you have a Learner Token.
-1. Make sure that your wallet is connected.
+2. Make sure that your wallet is connected.
 3. Press the **Commit to Assignment** above!
 4. You will be prompted to enter some **Assignment Info**. For now, you can just write a short message. Later in this course, you will learn more about how Assignment evidence works.
 
 ## You will know you are successful if:
+
 1. Your Learner Token is no longer in your wallet.
 2. You can see the transaction you just sent in your transaction history.
 3. You can [see your Assignment Commitment listed here](/course-management/assignment-commitments).
```

Comment:
Would we have a production link at launch?

Reply:
I think this kind of relative link works in production. Checking now

Reply:
Ah - link is fine we just need to enable the course management features in the Mesh PBL deployment -- I'll add it.

Reply:
@jinglescode I have an environment variable to add to Vercel deployment. Will DM.